com.wireframesketcher.WireframeSketcher.json

```diff
@@ -0,0 +1,81 @@
+{
+  "id": "com.wireframesketcher.WireframeSketcher",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "46",
+  "sdk": "org.gnome.Sdk",
+  "sdk-extensions": [],
+  "command": "WireframeSketcher",
+  "separate-locales": false,
+  "tags": [
+    "proprietary"
+  ],
+  "rename-appdata-file": "wireframesketcherstudio.appdata.xml",
+  "rename-desktop-file": "wireframesketcherstudio.desktop",
+  "rename-icon": "WireframeSketcher",
+  "finish-args": [
+    "--require-version=1.7.1",
+    "--filesystem=host",
+    "--filesystem=/tmp:rw",
+    "--persist=.eclipse",
+    "--share=network",
+    "--share=ipc",
+    "--socket=x11",
+    "--allow=devel",
+    "--socket=session-bus",
+    "--device=dri"
+  ],
+  "modules": [
+    {
+      "name": "sketcher",
+      "buildsystem": "simple",
+      "build-options": {
+        "no-debuginfo": true
+      },
+      "build-commands": [
+      	". ${FLATPAK_BUILDER_BUILDDIR}/build.sh"
+      ],
+      "sources": [
+        {
+          "type": "file",
+          "path": "apply_extra.sh",
+          "dest-filename": "apply_extra"
+        },
+        {
+          "type": "file",
+          "path": "build.sh"
+        },
+        {
+          "type": "extra-data",
+          "filename": "sketcher.deb",
+          "url": "http://cdn.wireframesketcher.com/studio/dist/WireframeSketcher-7.2.1_amd64.deb",
+          "only-arches": [
+            "x86_64"
+          ],
+          "sha256": "aa4c543d93f31af61f282569c5baf9ba9fa4fefc3afe9497cd8f594bd539af8d",
+          "size": 127627220
+        },
+        {
+          "type": "file",
+          "dest-filename": "sketcher.deb",
+          "url": "http://cdn.wireframesketcher.com/studio/dist/WireframeSketcher-7.2.1_amd64.deb",
```

Comment:
why is this repeated twice, once as `extra-data` and once as a regular `file`

Reply:
File is used to extract appdata, dekstop and icon files during the build. extra-data is used to avoid redistributing the binary. I've used com.axosoft.GitKraken application as an inspiration for this method.

Reply:
are you the upstream developer ? why does it need to be extra data ?

Reply:
No don't do this, extra-data is supposed to be used only when upstream does not allow redistribution.

Reply:
Removed extra-data module